docs/DeveloperGuide.md

```diff
@@ -247,16 +247,40 @@ _{Explain here how the data archiving feature will be implemented}_
+| `***`  | User                                         | delete orders for the day                                          | I can remove unnecessary orders from my database                                                
+| `***`  | User                                         | list all my orders for the day                                     | I can view what orders I have to fulfil for the day                                          |
+| `***`  | User                                         | retrieve data stored in the previous sessions                      | (obvious)                                                                            |
+| `***`  | User logging in after a long time            | delete all orders, reset the application                           | I can start on a clean slate since the previous orders are now meaningless to me             |
+| `**`    | Advanced user                                | set up shortcuts and shortforms                                    | I can reduce the time it takes to enter orders.                                              |
+| `**`    | User                                         | receive reminders for the orders due soon                          | I remember and complete them on time.                                                        |
+| `**`    | User who's at least slightly experienced     | delete multiple orders at one go                                   | I can track mass cancellations if they occur                                                 |
+| `**`    | User                                         | sort the orders according to which one needs to be completed first | I can prioritise the orders to work on for that day/                                         |
+| `**`    | User that works a lot at night               | set the application to dark mode                                   | My eyes remain healthy                                                                       |
+| `**`    | User                                         | add recipes                                                        | I know what ingredients are required for a particular order                                  |
+| `**`    | User                                         | edit individual orders                                             | I can update orders if a customer changes it instead of deleting and readding another order. |
+| `**`    | User                                         | add notes and special requests for orders                          | Details on customized orders can be mentioned together with the main order.                  |
+| `**`    | User who loves stats                         | View my most ordered products                                      | I know what to promote more                                                                  |
```

Comment:
Noticed you used capital "V" for "View". Maybe standardize and use small caps. Same for the benefits part in line 264 and 267 "My" and "Details".